app/views/layouts/application.html.haml

```diff
@@ -4,11 +4,13 @@
     %meta{ content: 'text/html; charset=UTF-8', 'http-equiv': 'Content-Type' }/
     %title
       = yield(:page_title) || 'Report management information to CCS'
-
     %meta{ name: 'viewport', content: 'width=device-width, initial-scale=1' }/
     %meta{ 'http-equiv': 'x-ua-compatible', content: 'ie-edge' }/
     %meta{ name: 'format-detection', content: 'telephone=no' }/
 
+    = favicon_link_tag('favicon.ico')
+    = favicon_link_tag('govuk-apple-touch-icon.png', rel: 'apple-touch-icon', type: 'image/png')
```

Comment:
As above, use the full path to the asset:

```suggestion
    = favicon_link_tag('govuk-frontend/assets/images/govuk-apple-touch-icon.png', rel: 'apple-touch-icon', type: 'image/png')
```

Reply:
@tekin how did you check this with `config.assets.compile = false` I then run into css and js issues?

Reply:
1. `bundle exec rake assets:compile`
2. set `config.assets.compile = false`
3. (re)start the rails server

Reply:
Oh, you may also need to set `config.assets.debug = false`!